app/models/preference.rb

```diff
@@ -1,6 +1,6 @@
 # Expresses someone's availability for a specific day/time
 class Preference < ApplicationRecord
-  validates :preference_type, presence: true
+  validates_presence_of :preference_type
```

Comment:
`validates_presence_of` is the old version, right? Shouldn't we be using `validates :preference_type, presence: true` ?